bridgetown-core/lib/bridgetown-core/concerns/site/content.rb

```diff
@@ -88,9 +137,8 @@ def collection_names
       end
     end
 
-    # Get all the documents
-    #
-    # Returns an Array of all Documents
+    # An Array of collection names
+    # @return [Array<String>] an array of collection names from the configuration
     def documents
```

Comment:
Oops, got the wrong description…it copied the `collection_names` one from above.